packages/icons/icons.json

```
@@ -4184,5 +4184,20 @@
         "tags": "footnote, notebook, note, reference, text, citation",
         "group": "editor",
         "codepoint": 62294
+    },
+    {
+        "displayName": "Binary",
+        "iconName": "binary",
+        "tags": "data, type, number, numeric, format",
+        "group": "miscellaneous",
```

I think this new icon belongs in the "data" group.

```suggestion
        "group": "data",
```

existing icons like `"floating-point"` also should move to the "data" group, but that can happen in a separate PR